src/ConfigEditor.tsx

```diff
@@ -9,19 +9,25 @@ import React from 'react';
 import { JWTConfigEditor } from './components/JWTConfigEditor';
 import { JWTForm } from './components/JWTForm';
 import { ConfigurationHelp } from './components/ConfigurationHelp';
-import { DEFAULT_REGION, GOOGLE_AUTH_TYPE_OPTIONS, PROCESSING_LOCATIONS, QUERY_PRIORITIES } from './constants';
+import { GOOGLE_AUTH_TYPE_OPTIONS, PROCESSING_LOCATIONS, QUERY_PRIORITIES } from './constants';
 import { BigQueryOptions, BigQuerySecureJsonData, GoogleAuthType, QueryPriority } from './types';
-import { getApiClient } from './api';
-
-import { DatasetSelector } from 'components/DatasetSelector';
 
 export type BigQueryConfigEditorProps = DataSourcePluginOptionsEditorProps<BigQueryOptions, BigQuerySecureJsonData>;
 
 export const BigQueryConfigEditor: React.FC<BigQueryConfigEditorProps> = (props) => {
   const { options, onOptionsChange } = props;
   const { jsonData, secureJsonFields, secureJsonData } = options;
+
```

Comment:
Would love to see this file in an npm package that we could use in CloudMonitoring!

Reply:
We'll create such. It can be used in the sheets ds as well